azure-arc-for-servers/README.md

```diff
@@ -0,0 +1,47 @@
+# Managing Credentials on Azure ARC for Servers
+
+Azure ARC, announced at [Microsoft Ignite 2019][Ignite], extends the Azure management capabilities to infrastructures across on-premises, multi-cloud, and edge devices. Azure ARC for Servers is one of Azure ARC technologies that enable you to manage your on-prem servers  through a unified Azure control plane such as Azure Portal, PowerShell cmdlet, REST API as if they were running in Azure IaaS.  See [here for more details][overview].
+
+## Connecting Your Machine to Azure
+You need to download the agent package and install it on your machine, which can be your desktop, laptop, or virtual machines. It works on Windows or Linux OSs. 
+Please see [this QuickStart guidance for the setup][quickstart].
+After connect your machine to Azure (`called connected machine` here), 
+you can perform management tasks as described in the [Azure ARC document][scenario] including assigning Azure policies for security compliance, monitoring your machine using Log Analysis, tagging your machines etc. 
+Apart from that, you may want to develop an application that runs on the connected machines but needs to connect to your Azure services. As we know, one of the challenges while building cloud applications is to manage the credentials because they cannot be checked-in as a part of your code or stored in your dev box. Fortunately, Azure ARC for Servers makes it easy for you. 
+
+## Acquiring Azure Access Token
+On the connected machine, you can request Azure Hybrid Instance Metadata Service (himds) for Azure access token. 
+Once your app gets a token, it can be connected to your Azure services by passing the token in the Authorization header using the Bearer scheme as if from an Azure VM.
+For example, assuming your app needs to connect to your Azure Cosmos DB, first you need to retrieve the connection string and other secrets stored in Azure Key Vault. The [sample code][kv] illustrates how to retrieve Azure access token from on ARC servers. 
+Thanks to [PowerShell 7 Preview 6 release][ps6], we can now use "Skip Error Check for Web Cmdlets" feature as shown in the example.
```

Comment:
Do we want to take a dependency on PS7-preview?

Reply:
yes. Added it in the readme